Formula/x/xprinter.rb

```diff
@@ -0,0 +1,19 @@
+class Xprinter < Formula
+  desc "Driver of xprinter"
+  homepage "https://galaxycal.github.io/homebrew-xprinter/"
+  url "https://github.com/GalaxyCal/homebrew-xprinter/releases/download/v1.0.0/xprinter-1.0.0.tar.xz"
+  sha256 "2bacdf67ca39448f2cf78a79f9a58c3ba0c070dd111fb90c4496c28d8572ff95"
+
+  depends_on :macos
+
+  def install
+    on_macos do
+      bin.install "xprinter"
+      bin.install "xprinter-server"
+    end
+  end
+
+  test do
+    system "#{bin}/xprinter", "--version"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.